vision.md

```diff
@@ -0,0 +1,39 @@
+# Vision of the buildfarmer payload
+Subset of tasks defined to increase the quality of the ROS/Gazebo code by actively tracking and categorizing problems shipped with new code in a timely manner, making it easier for the development teams to:
+* Fix most errors when they are introduced.
+* Decide if an error that pops up in CI is a flaky test already existing in the codebase.
+* Identify how often an error occurs and decide how to proceed.
+
+## Internal process and rationale behind it
+
+We already have a priority document explaining the order of relevancy of the possible errors, this is a new set of standards to define what we expect of the internal buildfarmer process:
+
+1. Auto-log any error appearing in the nightlies, even when there is no one around. This could serve future investigations.
+2. Any person can become a Buildfarmer Payload in case of necessity, this implies:
+   * Basic duties should be easy to automate, even scripted.
+   * Every Buildfarmer Payload must log/communicate internally any actions completed in the name of the role, so other folks can make progress on top of previous actions.
```

Comment:
Same here it's odd saying payload when referencing a  person. Either it should be a buildfarmer or perform buildfarmer payload